src/webview/html/messageTypingAsHtml.js

```diff
@@ -18,8 +18,10 @@ const typingAvatar = (realm: string, user: User): string => template`
 export default (realm: string, users: User[]): string => template`
   $!${users.map(user => typingAvatar(realm, user)).join('')}
   <div class="content">
-    <span></span>
-    <span></span>
-    <span></span>
+    <div dir="auto" class="message-content">
```

Comment:
This is our typing indicator. The only way to have RTL here is if the language selected is RTL... I think this should be grouped with a potential rework of the whole UI to be RTL if the language setting dictates this.